lib/ansible/modules/packaging/os/zypper.py

```diff
@@ -107,6 +108,18 @@
         required: false
         default: "no"
         choices: [ "yes", "no" ]
+    retries:
+        version_added: "2.3"
+        description:
+            - Number of retries if another instance of zypper is running. Defaults to no retries.
+        required: false
+        default: 0
+    retry_interval:
+        version_added: "2.3"
+        description:
+            - Time in seconds to wait between retries.
+        required: false
+        default: 10
```

Comment:
i just realize this was already pointed out, but I would still like a response to it